File: ClientApp/src/app/views/basic-setup/model/leave-type.ts
export class LeaveType {
    leaveTypeId: number;
    leaveTypeName: string;
    shortName: string;
    isCarryForward : boolean 
    isActive: boolean|null;
    elWorkDayCal: boolean;
    showReport: boolean
    remark: string

    constructor() {
        this.leaveTypeId = 0;
        this.leaveTypeName = "";
        this.shortName = "";
        this.isActive = null;
        this.elWorkDayCal = true;
        this.showReport = false;
        this.remark = "";
        this.isCarryForward = false
    }
}

File: Hrm.Domain/LeaveType.cs
using Hrm.Domain.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hrm.Domain
{
    public class LeaveType: BaseDomainEntity
    {
        public int LeaveTypeId { get; set; }
        public string LeaveTypeName { get; set; }
        public string? ShortName { get; set; }
        public bool? ELWorkDayCal {  get; set; }
        public bool IsActive { get; set; }
        public bool? ShowReport { get; set; }
        public string? Remark { get; set; }
        public int? MenuPosition { get; set; }
        public bool? IsCarryForward { get; set; }

        public virtual ICollection<LeaveRules> LeaveRules { get; } = new List<LeaveRules>();
        public virtual ICollection<LeaveRequest> LeaveRequests {get; } = new List<LeaveRequest>();
    }
}
